marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/ModelController.java

```diff
@@ -46,6 +52,50 @@
 @Controller
 @RequestMapping("/api/models")
 public class ModelController extends BaseController {
+    ObjectMapper objectMapper = new ObjectMapper();
+    @RequestMapping(value = "/hubCentralConfig", method = RequestMethod.GET)
+    @ResponseBody
+    @ApiOperation("This retrieves the Hub Central Config for models")
+    public ResponseEntity<?> getHubCentralConfig() throws FailedRequestException, ResourceNotFoundException, ForbiddenUserException {
+        return ResponseEntity.ok(getHubCentralConfigObject());
```

Comment:
Shouldn't we return an empty config if there is no configuration yet?

Reply:
Yes. The latest push handles it that way.